Could not load class 'org.opennms.netmgt.dao.support.IndexStorageStrategy'

Datacollection config doesn't work and stops the starting process of OpenNMS.

```
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 
'resourceDao' defined in class path resource [META-INF/opennms/applicationContext-dao.xml]: 
Invocation of init method failed; nested exception is 
org.springframework.orm.ObjectRetrievalFailureException: Could not load class 
'org.opennms.netmgt.dao.support.IndexStorageStrategy' for resource type 'sinfCpuInstance'; nested 
exception is java.lang.ClassNotFoundException: org.opennms.netmgt.dao.support.IndexStorageStrategy
```

--- a/datacollection/snmp-informant-standard.dc.xml
+++ b/datacollection/snmp-informant-standard.dc.xml
@@ -4,20 +4,20 @@
     <!-- SNMP Informant Resource Types -->
     <resourceType name="sinfCpuInstance" label="SNMP Informant CPU"
                   resourceLabel="CPU${sinfCpuInstance}">
-        <persistenceSelectorStrategy class="org.opennms.netmgt.collectd.PersistAllSelectorStrategy"/>
-        <storageStrategy class="org.opennms.netmgt.dao.support.IndexStorageStrategy"/>
+        <persistenceSelectorStrategy class="org.opennms.netmgt.collection.support.PersistAllSelectorStrategy"/>
+        <storageStrategy class="org.opennms.netmgt.collection.support.IndexStorageStrategy"/>
     </resourceType>
 
     <resourceType name="sinfLDskInstance" label="SNMP Informant Logical Disk"
                   resourceLabel="${sinfLDskInstance}">
-        <persistenceSelectorStrategy class="org.opennms.netmgt.collectd.PersistAllSelectorStrategy"/>
-        <storageStrategy class="org.opennms.netmgt.dao.support.IndexStorageStrategy"/>
+        <persistenceSelectorStrategy class="org.opennms.netmgt.collection.support.PersistAllSelectorStrategy"/>
+        <storageStrategy class="org.opennms.netmgt.collection.support.IndexStorageStrategy"/>
     </resourceType>
 
     <resourceType name="sinfNetInstance" label="SNMP Informant Network Interface"
                   resourceLabel="${sinfNetInstance}">
-        <persistenceSelectorStrategy class="org.opennms.netmgt.collectd.PersistAllSelectorStrategy"/>
-        <storageStrategy class="org.opennms.netmgt.dao.support.IndexStorageStrategy"/>
+        <persistenceSelectorStrategy class="org.opennms.netmgt.collection.support.PersistAllSelectorStrategy"/>
+        <storageStrategy class="org.opennms.netmgt.collection.support.IndexStorageStrategy"/>
     </resourceType>
 
     <group name="snmpinformant-memory" ifType="ignore">

